src/pages/search-help.vue

```diff
@@ -202,17 +222,29 @@
   </VContentPage>
 </template>
 
-<script>
+<script lang="ts">
+import { defineComponent, useContext } from '@nuxtjs/composition-api'
+
 import VLink from '~/components/VLink.vue'
 import VContentPage from '~/components/VContentPage.vue'
 
-const SearchHelpPage = {
-  name: 'search-help-page',
+const SearchHelpPage = defineComponent({
+  name: 'SearchHelpPage',
   components: { VLink, VContentPage },
-  methods: {
-    providerSearchLink(providerCode) {
+  setup() {
+    const { app } = useContext()
+    const pathFromQuery = (queryString: string, quote = false) => {
+      return app.localePath({
+        path: 'search',
+        query: {
+          q: quote ? `"${queryString}"` : queryString,
+        },
+      })
+    }
```

Comment:
Love this util. I'm not sure if it's used elsewhere but it's nice to not hardcode stuff.